@stellar/typescript-wallet-sdk/src/walletSdk/Auth/index.ts

```diff
@@ -165,3 +192,22 @@ const validateToken = (token: string) => {
     throw new ExpiredTokenError(parsedToken.expiresAt);
   }
 };
+
+const createAuthSignToken = async (
+  account: AccountKeypair,
+  claims: AuthHeaderClaims,
+  clientDomain?: string,
+  authHeaderSigner?: AuthHeaderSigner,
+) => {
+  if (!authHeaderSigner) {
+    return null;
+  }
+
+  const issuer = clientDomain ? null : account;
```

Comment:
@Ifropc just want to make sure this is right ([from kotlin](https://github.com/stellar/kotlin-wallet-sdk/blob/63b6e1bd2d93c2ce2b87cb4396d4025ffc87cf72/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/auth/Sep10.kt#L211))

if the clientDomain is given, then issuer is null. That's right?

Reply:
Yeah, for non-custodial wallets they need to set issuer equal to the domain account